frontend_tests/node_tests/settings_user_groups.js

```diff
@@ -90,12 +90,12 @@ run_test("can_edit", () => {
     assert(settings_user_groups.can_edit(1));
 });
 
-const user_group_selector = "#user-groups #1";
-const cancel_selector = "#user-groups #1 .save-status.btn-danger";
-const saved_selector = "#user-groups #1 .save-status.sea-green";
-const name_selector = "#user-groups #1 .name";
-const description_selector = "#user-groups #1 .description";
-const instructions_selector = "#user-groups #1 .save-instructions";
```

Comment:
I would leave out `CSS.escape` here.  I also wonder what @andersk thinks we should do with the real code here.  I would probably prefer `data-user-group-id=1` to `id=1` in the real code.  (I know that's out of scope of the current PR, but we could make a quick issue to clean this up.)

Reply:
cc @aero31aero in case he gets to User Group settings soon as part of general settings cleanup

Reply:
Agreed that this is an appalling misuse of `id`, but since we [can’t leave out `CSS.escape`](https://github.com/zulip/zulip/pull/17200#discussion_r570444414) either way, we can fix that equally well before or after this PR.